src/app/admin/collectionCentresTable/CollectionCentresTable.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+
+import React, { useState } from "react";
+import Table, { TableHeaders } from "@/components/Tables/Table";
+import styled from "styled-components";
+import Modal from "@/components/Modal/Modal";
+import Button from "@mui/material/Button";
+import DeleteIcon from "@mui/icons-material/Delete";
+import RefreshPageButton from "@/app/admin/RefreshPageButton";
+import { Schema } from "@/database_utils";
+import supabase from "@/supabaseClient";
+
+const DangerDialog = styled(Modal)`
+    & #deleteCollectionCentreDialog {
+        background-color: ${(props) => props.theme.error};
+    }
+`;
+
+export const OptionButtonDiv = styled.div`
+    display: flex;
+    padding-top: 1rem;
+    gap: 1rem;
+    justify-content: center;
+`;
+
+const collectionCentresTableHeaderKeysAndLabels: TableHeaders = [
+    ["name", "Name"],
+    ["acronym", "Acronym"],
+];
+
+interface Props {
+    collectionCentreData: Schema["collection_centres"][];
```

Comment:
Can't this just be a string array? Or no?

Reply:
No
`Schema["collection_centres"]` is `{name: string; acronym: string}`

Reply:
Also even if so, I think it would be more specific to say that the data is the one fetched from the database (otherwise if there are future updates from the database side on the Schema it might be type incompatible)

Reply:
Yeah but at that point I think we edit the code anyway. I think we should edit the type to what you put above as an interface.

Reply:
I think I agree with using an interface too